webapp/dependencies.js

```diff
@@ -2244,3 +2246,27 @@ function update_self_switch_values(graph, cell){
     graph.refresh();
 }
 
+//to get hex_color from colormap values
+function get_hex_color_code(block_uid, colormap_array){
+    var hex_color_arry = [];
+    var arr_len = colormap_array.length;
+    var counter_len = 0 ;
+    if(arr_len != 0 || arr_len != null){
+        counter_len = arr_len/3;
+    }
+    for(var i = 0; i < counter_len; i++){
+        var r = Math.round(colormap_array[i]*255);
+        var g = Math.round(colormap_array[counter_len + i]*255);
+        var b = Math.round(colormap_array[(counter_len*2) + i]*255);
```

Comment:
```suggestion
        var b = Math.round(colormap_array[counter_len*2 + i]*255);
```
remove inner brackets here. not required.